lib/graphql/schema/printer.rb

```diff
@@ -84,10 +84,62 @@ def print_deprecated(field_or_enum_value)
           end
         end
 
+        module DescriptionPrinter
+          def print_description(definition, indentation='', first_in_block=true)
+            return '' unless definition.description
+
+            lines = definition.description.split("\n")
+
+            description = indentation && !first_in_block ? "\n" : ""
+
+            lines.each do |line|
+              if line == ''
+                description << "#{indentation}#\n"
+              else
+                sublines = break_line(line, 120 - indentation.length)
+                sublines.each do |subline|
+                  description << "#{indentation}# #{subline}\n"
+                end
+              end
+            end
+
+            description
+          end
+
+          def break_line(line, length)
+            return [line] if line.length < length + 5
+
+            parts = line.split(Regexp.new("((?: |^).{15,#{length - 40}}(?= |$))"))
```

Comment:
Can you help me understand this regexp? I'm certainly not opposed to regexps but I can't quite grok this one 😬 

Maybe its ... trying to find natural line break positions?